backend/engine/plugins/github_repo_health/cli/src/utilities/config.py

```diff
@@ -9,12 +9,13 @@
     "name": "default",
     "version": "0.0.1",
     "rules": [
```

Comment:
Is the addition of branch rulesets to the default config planned for a later update to the feature branch (once rules can be joined together)?

Reply:
Yup! Once we add composite rules, we'll add composite rules that mimic the functionality of the original rules, but will succeed for either branch protection rules or branch rules.

For example, we'll have a composite rule, `branch_commit_signing`, that succeeds if `branch_protection_commit_signing` or `branch_rule_commit_signing` succeeds (and the same for the other branch protection rules)